src/controllers/autoCompletion.controller.js

```diff
@@ -0,0 +1,25 @@
+const Query = require("../models/Query");
+
+exports.getAutoCompletions = async function (req, res, next) {
+  const histories = [];
+  const userInput = req.query.keywords;
```

Comment:
```suggestion
  const { userInput } = req.query;
```
axios params를 어떻게 쓸지 이야기 해보면 좋을 것 같습니다.프론트 요청부분도 수정해서 다음과 같이 구조분해 할당으로 만드는게 나을 것 같은데 어떡해 생각하시나요?

Reply:
좋습니다! `userInput`으로 수정하고 구조분해 할당으로 통일하면 좋은 것 같습니다!